docs/file-source/environment-variables.md

```diff
@@ -10,6 +10,7 @@ PRIVACY_POLICY_URL=xxx
 WEB_CLIENT_ID=xxx  
 SIZE_MATTERS_BASE_WIDTH=375  
 SIZE_MATTERS_BASE_HEIGHT=670
+SERVICE_ACCOUNT_PATH=`serviceAccount-adminsdk.json` 
```

Comment:
A code block would be helpful to display these variables. Use three consecutive backticks (` ``` `) followed by the language for syntax highlighting:
**Input:**
````markdown
```
API_KEY=xxx  
AUTH_DOMAIN=xxx  
DATABASE_URL=xxx  
PROJECT_ID=xxx  
STORAGE_BUCKET=xxx  
APP_ID=xxx  
PRIVACY_POLICY_URL=xxx  
WEB_CLIENT_ID=xxx  
SIZE_MATTERS_BASE_WIDTH=375  
SIZE_MATTERS_BASE_HEIGHT=670
SERVICE_ACCOUNT_PATH=`serviceAccount-adminsdk.json` 
```
````
**Output:**
```
API_KEY=xxx  
AUTH_DOMAIN=xxx  
DATABASE_URL=xxx  
PROJECT_ID=xxx  
STORAGE_BUCKET=xxx  
APP_ID=xxx  
PRIVACY_POLICY_URL=xxx  
WEB_CLIENT_ID=xxx  
SIZE_MATTERS_BASE_WIDTH=375  
SIZE_MATTERS_BASE_HEIGHT=670
SERVICE_ACCOUNT_PATH=`serviceAccount-adminsdk.json` 
```